src/handlers/payout/post.ts

```diff
@@ -86,29 +103,33 @@ export const incentivizeCreatorComment = async () => {
   }
   const context = getBotContext();
   const payload = context.payload as Payload;
-  const org = payload.organization?.login;
   const issue = payload.issue;
-  if (!issue || !org) {
-    logger.info(`Incomplete payload. issue: ${issue}, org: ${org}`);
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
     return;
   }
-  const assignees = issue?.assignees ?? [];
+
+  const assignees = issue.assignees ?? [];
```

Comment:
```suggestion
  const assignees = issue.assignees || [];
```

Should it be `or`?

Reply:
`||` will return the right element if the left is falsy, but `??` will only return right element if the left is `undefined` or `null`. In this context it doesn't really matter.